omnibus/files/server-ctl-cookbooks/infra-server/recipes/oc-chef-pedant.rb

```diff
@@ -62,7 +62,7 @@
     search_engine_url: OmnibusHelper.new(node).search_engine_url,
     search_provider: node['private_chef']['opscode-erchef']['search_provider'],
     search_auth_username: node['private_chef']['opscode-erchef']['search_auth_username'],
-    search_auth_password: node['private_chef']['opscode-erchef']['search_auth_password'],
+    search_auth_password: node['private_chef']['opscode-erchef']['search_auth_password'] || PrivateChef.credentials.get('opscode_erchef', 'search_auth_password'),
```

Comment:
Would the rest of the params not see a similar issue?

Reply:
We are using defaults to search_auth_username in default.rb but not for search_auth_password.
https://github.com/chef/chef-server/blob/zei/shahid/internal_opensearch/omnibus/files/server-ctl-cookbooks/infra-server/attributes/default.rb#L356-L357